wgpu/src/dispatch.rs

```diff
@@ -795,55 +782,143 @@ macro_rules! dispatch_types_inner {
     };
 }
 
-macro_rules! dispatch_types {
-    (
-        wgpu_core = $wgpu_core_context:ty;
-        webgpu = $webgpu_context:ty;
-        {$(
-            $type:tt;
-        )*}
-    ) => {
-        $(
-            dispatch_types_inner!{
-                wgpu_core = backend::ContextWgpuCore;
-                webgpu = backend::ContextWebGpu;
-                $type;
-            }
-        )*
+dispatch_types_inner! {
+    {
+        ref type DispatchInstance = Instance:InstanceInterface
     };
 }
```

Comment:
can this syntax be made more terse, now that there's only ever one being done per macro invocation?

Reply:
Yes we can! Done in https://github.com/gfx-rs/wgpu/pull/6658/commits/bd92b353099ed393405ce7a65cec0e0a7085129a